src/api/overTheBox/overTheBox.v6.service.js

```diff
@@ -109,6 +109,20 @@ angular.module('ovh-api-services').service('OvhApiOverTheBoxV6', ($resource, Ovh
       url: '/overTheBox/:serviceName/device/logs ',
       isArray: false,
     },
+    getActions: {
+      method: 'GET',
+      url: '/overTheBox/:serviceName/device/actions',
+      isArray: true,
+    },
+    getAction: {
+      method: 'GET',
+      url: '/overTheBox/:serviceName/device/actions/:actionId',
+    },
+    unlinkDevice: {
+      method: 'DELETE',
+      url: '/overTheBox/:serviceName/device',
+      interceptor,
```

Comment:
This looks like it could be in a `device` submodule

Reply:
you're right

Reply:
Maybe the endpoints declaration could be removed in this file